cli/services/helm.go

```diff
@@ -0,0 +1,83 @@
+package services
+
+import (
+	"errors"
+
+	"github.com/elastic/metricbeat-tests-poc/cli/shell"
+	log "github.com/sirupsen/logrus"
+)
+
+// HelmManager defines the operations for Helm
+type HelmManager interface {
+	AddRepo(repo string, URL string) error
+	DeleteChart(chart string) error
+	InstallChart(name string, chart string, version string) error
+}
+
+// HelmFactory returns oone of the Helm supported versions, or an error
+func HelmFactory(version string) (HelmManager, error) {
+	var helm HelmManager
+	if version == "3.1.1" {
+		return &helm3_1_1{}, nil
+	}
+
+	return helm, errors.New("Sorry, we don't support Helm v" + version + " version")
+}
+
+type helm3_1_1 struct {
```

Comment:
@kuisathaverat Creating a struct for Helm v2 we will support different command execution.

Please let me know if you can help me with the commands

Reply:
sure